src/main/webapp/scripts/calendar-script.js

```diff
@@ -0,0 +1,190 @@
+  const description = event.description;
+
+  const listElement = document.createElement('LI');
+  const leftDiv = document.createElement('div');
+
+  // summary
+  const heading = document.createElement('h3');
+  heading.appendChild(document.createTextNode(summary));
+  leftDiv.appendChild(heading);
+
+  // Time
+  const monthNames = ['January', 'February', 'March', 'April', 'May',
+    'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+  const timeNode = document.createElement('h4');
+  const start = new Date(startDateTime);
+  const end = new Date(endDateTime);
+  const startDate = monthNames[start.getMonth()] + ' ' +
+      start.getDate() + ', ' + start.getFullYear();
+  const endDate = monthNames[end.getMonth()] + ' ' +
+      end.getDate() + ', ' + end.getFullYear();
+
+  /**
+   * Gets the time string from the Date object
+   * @param {Date} dateTime Date from which we receive the time string from
+   * @return {String} String which represents time
+   */
+  function getTime(dateTime) {
+    let time = '';
+    if ((dateTime.getHours() % 12) == 0) {
+      time += '12';
+    } else {
+      time += dateTime.getHours() % 12;
+    }
+    time += ':';
+    if (dateTime.getMinutes() < 10) {
+      time += '0' + dateTime.getMinutes();
+    } else {
+      time+=dateTime.getMinutes();
+    }
+    return time;
+  }
+
+  const startTime = getTime(start);
+  const endTime = getTime(end);
+
+  if (startDate == endDate) {
+    timeNode.appendChild(
+        document.createTextNode(
+            startDate+ ' ' + startTime + ' - ' + endTime));
+  }
+  leftDiv.appendChild(timeNode);
+
+
+  // description
+  const descriptionNode = document.createElement('p');
+  if (description) {
+    descriptionNode.appendChild(document.createTextNode(description));
+  } else {
+    descriptionNode.appendChild(
+        document.createTextNode('No Description Available'));
+  }
+  leftDiv.appendChild(descriptionNode);
+  leftDiv.classList.add('summary');
+
+  listElement.append(leftDiv);
+
+  // button
+  const button = document.createElement('button');
+  button.appendChild(document.createTextNode('Add to Calendar'));
+  button.addEventListener('click', function() {
+    addEventToCalendar(event, 'primary');
+    this.disabled = true;
+    this.innerText='Event Added!';
+  });
+
+  getPrimaryIds().then(function(primaryIds) {
+    if (primaryIds.has(event.id)) {
+      gapi.client.calendar.events.get({
+        'calendarId': 'primary',
+        'eventId': event.id,
+      }).execute(function(e) {
+        if (e.status == 'confirmed') {
+          button.disabled = true;
+          button.innerText = 'Event Added!';
+        }
+      });
+    }
+  }).then(function() {
+    listElement.appendChild(button);
+  });
+  $('#open-events').append(listElement);
+}
+
+/**
+ * Adds an event to calendar with given id
+ * @param {Event} eventToAdd Event to be added
+ * @param {String} calendarID ID of calendar that is targeted
```

Comment:
so each even has a unique calendar ID?

Reply:
calendar ID is the ID of the calendar you wanna add the event to

Reply:
ah ok